modules/ROOT/pages/index-vm-bare-metal.adoc

```diff
@@ -45,7 +45,7 @@ Mule applications run on workers. Multiple replicas of applications can run acro
 
 The following diagram shows the general network architecture of Runtime Fabric on VMs / Bare Metal.
 
-image::architecture-network.png[]
+image::architecture-network.png["Diagram of network architecture of Runtime Fabric and TCP load balancer connection with internal load balancers of controllers and workers"]
 
 This diagram shows the TCP load balancer used to load balance requests across the internal load balancers running on the controllers. It also shows the internal load balancer that distributes requests to each replica of Mule applications running on the workers.
```

Comment:
When you describe the image in the surrounding text (best practice!), you don't need to repeat in alt text. In this case, if you have described every element of the diagram already in the surrounding text, then you can [""]. If there's something that's not mentioned in the surrounding text, put only that.